packages/material/scss/_variables.scss

```diff
@@ -84,13 +84,16 @@ $textarea-floating-label-padding: 25px !default;
 
 /// The horizontal padding of the container.
 /// @group toolbar
-$toolbar-padding-x: $padding-x !default;
+$toolbar-padding-x: 0 !default;
 
 /// The vertical padding of the container.
 /// @group toolbar
-$toolbar-padding-y: $padding-x !default;
-$toolbar-calc-size: #{$button-calc-size} + #{$toolbar-padding-y * 2} + 2px;
-$toolbar-inner-calc-size: #{$button-calc-size} + #{$toolbar-padding-y * 2};
+$toolbar-padding-y: 0 !default;
+$toolbar-button-padding-x: $button-padding-x !default;
+$toolbar-button-padding-y: $padding-y*2.5 !default;
+$toolbar-button-calc-size: #{$form-line-height-em} + #{$toolbar-button-padding-y * 2} + #{$button-border-width * 2};
+$toolbar-calc-size: #{$toolbar-button-calc-size} + #{$toolbar-padding-y * 2} !default;
```

Comment:
This variable is not used anywhere.

Reply:
this variable comes from the _variables in the Default theme